extensions/scheduler/api/src/main/java/io/quarkus/scheduler/Scheduler.java

```diff
@@ -57,4 +63,105 @@ public interface Scheduler {
      * @return the trigger of a specific job or null for non-existent identity.
      */
     Trigger getScheduledJob(String identity);
+
+    /**
+     * Creates a new trigger definition that can be used to schedule a task programmatically.
+     * <p>
+     * The properties of the trigger definition have the same semantics as their equivalents in the {@link Scheduled}
+     * annotation.
+     *
+     * @param identity
+     * @return a new trigger definition
+     * @see TriggerDefinition#schedule(Consumer)
+     * @see TriggerDefinition#schedule(Consumer)
+     */
+    TriggerDefinition newTrigger(String identity);
```

Comment:
Hm, I think that something like `scheduler.addJob().setCron("0/5 * * * * ?").execute(ec -> { ...})` is more readable...

Reply:
Or maybe even `scheduler.newJob().setCron("0/5 * * * * ?").setTask(ec -> { ...}).schedule()`.

Reply:
Done.